client/src/lib/firebase/auth.ts

```diff
@@ -23,10 +31,11 @@ export async function loginWithMicrosoft() {
   }
 }
 
-export async function loginWithEmail() {
+// Login with email and password
+export async function loginWithEmail(email: string, password: string) {
   try {
-    // const user = await createUserWithEmailAndPassword(auth, email, password);
-    // console.log(user);
+    const user = await createUserWithEmailAndPassword(auth, email, password);
+    console.log(user);
   } catch (error) {
     console.log(error);
```

Comment:
the error proboly shouldn't be cought here. personally I would just return createUserWithEmailAndPassword directly

Reply:
resolved